### DISCUSSION TITLE
Disney XD poland

### DISCUSSION THREAD

**Discussion by whyiyeet** [iptv-org/community#947] [Q&A]
### Channel ID

DisneyXD.pl

### Notes (optional)

i want to watch polish thingy

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by hdhdghfjebdhhd62535** [comment]
Here is the link http://178.219.128.68:64888/DISNXD

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/pl.m3u#L108-L109
